dali/python/nvidia/dali/ops.py

```diff
@@ -380,17 +380,6 @@ def __init__(self, **kwargs):
                 if not self._schema.IsDeprecatedArg(arg_name):
                     continue
                 meta = self._schema.DeprecatedArgMeta(arg_name)
-                new_name = meta['renamed_to']
```

Comment:
Do I understand it correctly that now Python only issues warnings, but the errors are left to the backend?

Reply:
Yes